CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 2.15.2
+
+### Grafana Mimir
+
+* [BUGFIX] Update module golang.org/x/net to v0.36.0 to address [CVE-2025-22870](https://nvd.nist.gov/vuln/detail/CVE-2025-22870). #10875
+* [BUGFIX] Update module github.com/golang-jwt/jwt/v5 to v5.2.2 to address [CVE-2025-30204](https://nvd.nist.gov/vuln/detail/CVE-2025-30204). #11045
```

Comment:
do we want to mention the rest of the changes too? we have an SLO for those which isn't expired yet


- [CVE-2025-27144 / GHSA-c6gw-w398-hv78 / GO-2025-3485](https://github.com/grafana/mimir/pull/11045)
- [GHSA-mh63-6h87-95cp / CVE-2025-22868 / GO-2025-3488](https://github.com/grafana/mimir/pull/10875)
- [CVE-2025-22870 / GHSA-qxp5-gwg8-xv66 / GO-2025-3503](https://github.com/grafana/mimir/pull/10875)